app/views/job_applications/index.html.haml

```diff
@@ -45,6 +45,15 @@
                 = job_application.find_or_create_interview.try(:time) || t('jobs.not_set')
               %td
                 = job_application.find_or_create_interview.try(:location) || t('jobs.not_set')
+      - if admission.appliable? and @current_user.similar_jobs_not_applied_to.any?
```

Comment:
Use &&, and might bite you in the ass as it has different precedence rules associated to it.